SwiftUIDZ/SwiftUIDZ/BigToggle.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  BigToggle.swift
+//  SwiftUIDZ
+//
+//  Created by Алена Панченко on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Большой переключатель
+struct BigToggle: ToggleStyle {
```

Comment:
```suggestion
struct BigToggleView: ToggleStyle {
```
Если это вью в конце пишем View если это стиль в конце пишем Style